model/src/gateway/payload/invite_create.rs

```diff
@@ -0,0 +1,21 @@
+use crate::{
+    id::{ChannelId, GuildId},
+    user::User,
+};
+
+#[cfg_attr(
+    feature = "serde-support",
+    derive(serde::Deserialize, serde::Serialize)
+)]
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+pub struct InviteCreate {
+    pub channel_id: ChannelId,
+    pub code: String,
+    pub created_at: String,
+    pub guild_id: GuildId,
+    pub inviter: User, // TODO: This is a partial user, need feedback on whether to make a new struct for it here, or use something else
```

Comment:
I think we have a partial user somewhere else as well have to have a look, but you could probably make a new struct

Reply:
We don't so I made it into a new struct